haskell-src/Concordium/Types/Queries.hs

```diff
@@ -122,12 +123,18 @@ $(deriveJSON defaultOptions{fieldLabelModifier = firstLower . dropWhile isLower}
 data BlockInfo = BlockInfo
     { -- |The block hash
       biBlockHash :: !BlockHash,
-      -- |The parent block hash
+      -- |The parent block hash. For a re-genesis block, this will be the terminal block of the
+      -- previous chain. For the initial genesis block, this will be the hash of the block itself.
       biBlockParent :: !BlockHash,
       -- |The last finalized block when this block was baked
       biBlockLastFinalized :: !BlockHash,
       -- |The height of this block
-      biBlockHeight :: !BlockHeight,
+      biBlockHeight :: !AbsoluteBlockHeight,
+      -- |The genesis index for this block. This counts the number of protocol updates that have
+      -- preceded this block.
+      biGenesisIndex :: !GenesisIndex,
+      -- |The height of this block relative to the (re)genesis block at its genesis index.
+      biLocalBlockHeight :: !BlockHeight,
```

Comment:
The naming in consensus status is using "current era". I think the same naming should be used in both places, otherwise we have to confuse the users and explain the differences.

Reply:
I've renamed it to `biEraBlockHeight`. I don't think it should be `biCurrentEraBlockHeight`, since it is relative to the block's own era, not the most recent one.